README.md

```diff
@@ -486,6 +486,7 @@ Services to securely store your Docker images.
 * [Rapid Dashboard](https://github.com/ozlerhakan/rapid) - A simple query dashboard to use Docker Remote API by [@ozlerhakan](https://github.com/ozlerhakan/)
 
 # Useful Resources
+* [Awesome Kubernetes](https://github.com/ramitsurana/awesome-kubernetes) by [@ramitsurana](https://github.com/ramitsurana)
```

Comment:
At the bottom of the `README.md` (in edit mode) there is those links that we can use as reference elsewhere. It's a Markdown feature. 
For example:
[docker]: https://github.com/docker

you can use this link as `[here is a the link to docker][docker]` instead of `[here is the link to docker](https://github.com/docker)`.
Like this we avoid to have duplicate links.